packages/ketcher-react/src/script/ketcher.ts

```diff
@@ -19,6 +19,7 @@ import {
   SupportedFormat,
   Struct
 } from 'ketcher-core'
+import { StructServiceOptions } from 'ketcher-core/src/infrastructure/services/struct/structService.types'
```

Comment:
I'm assuming it should be imported from 
`import { StructServiceOptions } from 'ketcher-core'`
right?

Reply:
fixed in [fix import](https://github.com/epam/ketcher/pull/318/commits/73ced73152f3d3e44052ec4c0e01ef84c12a7a68)